### PR TITLE
Automate Case Details and Fields Preselection for Influenza Cases #13184

### LINKED ISSUE #13184: Automate Case Details and Fields Preselection for Influenza Cases [1]
### Feature Description

Automate certain fields for influenza cases upon creation to improve data entry efficiency and accuracy. This includes pre-selecting certain case details (e.g., case classification, investigation status) and facility address population when available.

### Added Value/Benefit

This automation will reduce manual data entry, ensuring standardized case details and decreasing the chance of user error.

### Acceptance Criteria

- [ ] Automatically select "CONFIRMED CASE" for case classification, "PENDING" for investigation status, and "NO OUTCOME YET" for outcome.
- [ ] Pre-select "Home" for "Place of Stay" 
- [ ] ~Hide the “CASE DEFINITION” info button for influenza.~

### Implementation Details
- Which are the desired case classification available for Influenza? => all, as described in the document (Page 3 1st bulletpoint (all visible)
- 

### Mockups

_No response_

### Additional Information

- Pre-select "Home" for "Place of Stay" unless the address matches a known facility, in which case the facility should be filled automatically. - **further clarification details needed** => When we do have an address in the lab messages, that can be checked against the existing facilities, we create another ticket for it

_No response_

### PR DESCRIPTION
**Pull request by cazacmarin**
Fixes #13184

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13200

**Review by sergiupacurariu — CHANGES_REQUESTED** [review]

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java [hunk 1]
```diff
@@ -550,6 +551,7 @@ private void setNoneFacility() {
 		FacilityReferenceDto noFacilityRef = FacadeProvider.getFacilityFacade().getByUuid(FacilityDto.NONE_FACILITY_UUID).toReference();
 		facilityCombo.addItem(noFacilityRef);
 		facilityCombo.setValue(noFacilityRef);
+		facilityType.setRequired(false);
```
Comment (leventegal-she) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java:554]: why is this needed?

Comment (cazacmarin) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java:554]: Without this line, this field will be set as required for validation. It happens only from code change. And in this case, in Validation Form, even if Place of Stay is set to `Home`, app will ask us to introduce the Facility Type(as Required).

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13200

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13200
